apps/tactics-finder/worker.py

```
@@ -48,136 +51,178 @@ def clean_pgn(pgn: str) -> str:
+                    "directStart": "false",
                 },
                 "userId": user_id,
                 "setId": set_id,
                 "last_puzzle": last_puzzle_for_this_call,
-            })
-            response.raise_for_status()
-            print(f"Sent puzzle {i+1}/{len(puzzle_data_list)} for set {set_id}. Last puzzle for set: {last_puzzle_for_this_call}")
+            }
+
+            try:
+                response = requests.post(API_URL, json=payload)
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)